go/cmd/vtctldclient/command/onlineddl.go

```diff
@@ -140,6 +156,66 @@ func commandOnlineDDLCleanup(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func commandOnlineDDLComplete(cmd *cobra.Command, args []string) error {
+	keyspace := cmd.Flags().Arg(0)
+	uuid := cmd.Flags().Arg(1)
+
+	switch {
+	case strings.ToLower(uuid) == AllMigrationsIndicator:
+	case schema.IsOnlineDDLUUID(uuid):
+	default:
+		return fmt.Errorf("argument must be 'all' or a valid UUID. Got '%s'", uuid)
+	}
```

Comment:
don't have suggestion for a good name, but want to pull this duplicate check out to something like:

```go
func whatever(cmd *cobra.Command) (keyspace, uuid string, err error) {
	keyspace = cmd.Flags().Arg(0)
	uuid = cmd.Flags().Arg(1)

	switch {
	case strings.ToLower(uuid) == AllMigrationsIndicator:
	case schema.IsOnlineDDLUUID(uuid):
	default:
		return "", "", fmt.Errorf("argument must be 'all' or a valid UUID. Got '%s'", uuid)
	}
        return keyspace, uuid, nil
}

func commandOnlineDDLComplete(cmd *cobra.Command, args []string) error {
    keyspace, uuid, err := whatever(cmd)
    if err != nil { return err }
    cli.FinishedParsing(cmd)
    // rest of func
}
```

Reply:
Done